.github/workflows/reusable-check-release-branch.yaml

```diff
@@ -1,5 +1,8 @@
 name: Check release branch
 
+env:
+  RUNNER_OS: ${{ vars.RUNNER || ‘ubuntu-latest’ }} # Default value fallback
```

Comment:
Should we be using the same variable here? `vars.RUNNER_OS` and `env.RUNNER_OS`? It seems like that would make it more apparent that these variables are related.

Reply:
Sure, that makes sense to me

Reply:
Done 👍